v1/src/main/java/com/google/cloud/teleport/spanner/ImportTransform.java

```diff
@@ -660,7 +667,7 @@ public void processElement(ProcessContext c) {
                             if (earlyIndexCreateFlag.get()
                                 && ((createForeignKeyStatements.size()
                                         + createIndexStatements.size())
-                                    >= EARLY_INDEX_CREATE_THRESHOLD)) {
+                                    >= earlyIndexCreateThreshold.get())) {
```

Comment:
Reading the original internal bug it seems the FR is to disable early creation. Shall we define a special number (0) when it is set to 0 then early creation is disabled?